stdlib/SparseArrays/src/sparsematrix.jl

```diff
@@ -354,6 +354,11 @@ similar(S::SparseMatrixCSC, ::Type{TvNew}, ::Type{TiNew}, m::Integer) where {TvN
 similar(S::SparseMatrixCSC, ::Type{TvNew}, ::Type{TiNew}, m::Integer, n::Integer) where {TvNew,TiNew} =
     _sparsesimilar(S, TvNew, TiNew, (m, n))
 
+function sizehint!(S::SparseMatrixCSC, n::Integer)
+    sizehint!(S.rowval, min(n, length(S)))
+    sizehint!(S.nzval, min(n, length(S)))
```

Comment:
would be clearer to set `nhint = min(n, length(S))` and then pass `nhint` to both `sizehint!` calls